azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/redux/middleware/handler/CallingMiddlewareActionHandler.kt

```diff
@@ -901,6 +901,19 @@ internal class CallingMiddlewareActionHandlerImpl(
                     ) {
                         store.dispatch(CaptionsAction.StartRequested(localOptions.captionsOptions?.spokenLanguage ?: ""))
                     }
+
```

Comment:
There is already a `onAudioDeviceChangeRequested` that is setting correct route.
It is better to figure out why in the Skip setup screen that is not called. Or called to late/early?
Anyway this code does not belong to here in `subscribeCallInfoModelEventUpdate`

Reply:
Pavel lets have discussion when you have some time.